src/index.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const state = {
+  city: 'Boston',
+  temp: 60,
+};
+
+const increaseTemp = () => {
+  state.temp += 1;
+  updateTempInfo();
+};
+
+const decreaseTemp = () => {
+  state.temp -= 1;
+  updateTempInfo();
+};
+
+const updateTempInfo = () => {
+  const temperature = document.querySelector('#temp');
+  let landscape;
+  if (state.temp >= 80) {
+    temperature.className = 'red';
+    landscape = '☀️🏖☀️🏝☀️⛵️';
+  } else if (state.temp >= 70) {
+    temperature.className = 'orange';
+    landscape = '🏕🌤💐🏔🌤';
+  } else if (state.temp >= 60) {
+    temperature.className = 'yellow';
+    landscape = '🍃🌦🍃☔️💨🌦';
+  } else if (state.temp >= 50) {
+    temperature.className = 'green';
+    landscape = '☃️❄️🏂⛄️🌨🎄';
+  } else if (state.temp <= 49) {
+    temperature.className = 'aqua';
+    landscape = '🧊🧊🥶🧊🧊🥶';
+  }
+  temperature.textContent = state.temp;
+  const updatedLandscape = document.querySelector('#landscape-icons');
+  updatedLandscape.textContent = landscape;
+};
+
+const updateCityName = () => {
+  const cityName = document.querySelector('#city-name');
+  const cityInput = document.querySelector('#city-input');
+  state.city = cityInput.value;
+  cityName.textContent = state.city;
+};
+
+const KelvinToFarenheit = (kelvin) => {
+  return 1.8 * (kelvin - 273.15) + 32;
+};
+
+const getLatAndLong = () => {
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: state.city,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      const lat = response.data[0].lat;
+      const long = response.data[0].lon;
+      //   return { lat, long };
+      getWeatherForCity(lat, long);
```

Comment:
I see that you invoke a method called `getWeatherForCity`, which I assume is supposed to call the weather API using lat and lon, but you don't have such a method defined. 

Is line 65 supposed to call getTempForCity() instead?